extensions/amp-inline-gallery/1.0/thumbnails.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as Preact from '../../../src/preact';
+import {BaseCarousel} from '../../amp-base-carousel/1.0/base-carousel';
+import {CarouselContext} from '../../amp-base-carousel/1.0/carousel-context';
+import {Wrapper} from '../../../src/preact/component';
+import {cloneElement, toChildArray, useContext} from '../../../src/preact';
+
+/**
+ * @param {!JsonObject} props
+ * @return {PreactDef.Renderable}
+ */
+export function Thumbnails({loop = true, children, ...rest}) {
```

Comment:
Is `loop=true` a default from 0.1? It seems to me a bit unexpected for the thumbnails to be looping by default.

Reply:
Yes it is a default from 0.1, though I can't tell from prior documents why this is. Do you prefer to change the default to `false`?

Reply:
Changed this to `false`.